app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] Great job so far 👏🏻 However according to the project requirements, The application should ask for confirmation of user email when a user registers himself. You can check and make sure the  `:confirmable` is enabled in the user model (`user.rb` file). You should look something similar to what is below:

   ```ruby
   	devise :database_authenticatable, :registerable,
   	:recoverable, :rememberable, :validatable, :confirmable
   ```